src/hooks/useLaunchableMarkets.ts

```diff
@@ -71,6 +73,10 @@ export const useMetadataService = () => {
         }
       });
 
+      ASSETS_TO_REMOVE.forEach((asset) => {
```

Comment:
why not just never add them in the first place?

Reply:
hohoho you're right. will change